pkg/service/record/record.go

```diff
@@ -119,9 +131,13 @@ func (r *recorder) CaptureTraffic(path string, appCmd, appContainer, appNetwork
 	case <-stopper:
 		abortStopHooksForcefully = true
 		loadedHooks.Stop(false)
+		tele.RecordedTestSuite(dirName, testsTotal, mocksTotal)
```

Comment:
add a check if no testcase is recorded we shouldn't fire any event

Reply:
done